doc/nestml_language/index.rst

```diff
@@ -8,8 +8,6 @@ NESTML language documentation
    neurons_in_nestml
    synapses_in_nestml
 
-NESTML is a domain specific language for the specification of models. NESTML is a completely generic language, but is specifically tailored for the spiking neural network simulator `NEST <http://www.nest-simulator.org>`__. It has a concise syntax based on that of Python, which avoids clutter in the form of semicolons, curly braces or tags as known from other programming and description languages. Instead, it concentrates on domain concepts that help to efficiently create neuron and synapse models.
+NESTML is a domain specific language for the specification of models; typically, the individual neurons and synapses that make up a neural network. NESTML is a completely generic language for hybrid dynamical systems, that is, those that exhibit continuous-time dynamics (such as described by ordinary differential equations) as well as instantaneous events. It has a concise syntax based on that of Python, which avoids clutter in the form of semicolons, curly braces or tags as known from other programming and description languages. Instead, it concentrates on domain concepts that help to efficiently create neuron and synapse models.
```

Comment:
Possibly leave a note here that NESTML is agnostic of the simulation backend, such as the numerical update schemes, ODE solvers etc.